.github/workflows/test.yaml

```
@@ -87,3 +89,19 @@
           yarn vitest run --changed origin/${GITHUB_BASE_REF} --coverage --reporter=junit --coverage.reporter=json-summary --outputFile=./coverage/coverage-summary.json
       - name: Report coverage
         uses: davelosert/vitest-coverage-report-action@v2
+
+  skip-visual-regression-test:
+    name: skip test
+    runs-on: ubuntu-latest
+    if: github.event_name == 'pull_request' && github.event.pull_request.draft == false && ${{ toJSON(github.event.pull_request.assignees) == '[]' }} && ( ${{ startsWith(github.event.pull_request.title, 'build:') || startsWith(github.event.pull_request.title, 'bump') }} )
```

actionlint (1.7.4)

if: condition "github.event_name == 'pull_request' && github.event.pull_request.draft == false && ${{ toJSON(github.event.pull_request.assignees) == '[]' }} && ( ${{ startsWith(github.event.pull_request.title, 'build:') || startsWith(github.event.pull_request.title, 'bump') }} )" is always evaluated to true because extra characters are around ${{ }}

(if-cond)